### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by RDxR10**
Format correction, grammar fix

### CODE REVIEW

File: content/homomorphisms/en/homomorphisms.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ b^{a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_j} \text{ times}}
 $$
 
 $$
-b^{a_i +a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_i} \text{ times}}\cdot\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_j} \text{ times}}
+b^{a_i + a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_i} \text{ times}} \cdot \underbrace{b \cdot b \cdot \dots \cdot b} _{a_j \text{ times}}
```
Comment (RareSkills) [inline content/homomorphisms/en/homomorphisms.md]: can we undo the changes in this file? It's not clear how this changes the formula rendering.

File: content/homomorphisms/en/homomorphisms.md [hunk 2]
```diff
@@ -275,4 +275,4 @@ Homomorphisms are not necessarily bidirectional. They are only required to work
 
 If $\phi: A \rightarrow B$ is computationally difficult to invert, then $\phi$ homomorphically encrypts the elements of $A$. That means we can validate claims about computations in $A$ using elements in $B$.
 
-The good news is, we are done with our treatment of abstract algebra, and we now have a strong foundation to move on to [elliptic curves](https://www.rareskills.io/post/elliptic-curve-addition).
```
Comment (RareSkills) [inline content/homomorphisms/en/homomorphisms.md:278]: same here

**Review by RareSkills — CHANGES_REQUESTED** [review]
please see comment in homomorphisms.md, it's probably best to revert the changes there.

python-lagrange-interpolation.py is good, thanks!

**Comment by RareSkills** [comment]
@RDxR10 can you explain the latex changes in homomorphisms.md?

It isn't clear that this changes the formula rendering and seems to be creating an unnecessary diff.

**Comment by RDxR10** [comment]
TBH I'm not sure either. But I'm guessing that the parser is unable to interpret what's markdown and what's LATEX.

For the other one: `elliptic-curves-finite-fields.md`
1. `has_sqrtmod_prime_power` was imported twice.
2. `libnum` was used directly without importing (which is not required).

**Review by RareSkills — APPROVED** [review]

**Pull request merged**